lib/themes/dosomething/paraneue_dosomething/templates/campaign/partials/campaign-creator.tpl.php

```diff
@@ -23,7 +23,7 @@
         <img src="<?php print $picture['src_square']; ?>" />
       </div>
       <div class="__body">
-        <h4 class="__title heading -delta"><?php print $first_name; ?> <?php print $last_initial; ?></h4>
+        <h4 class="__title heading -delta"><?php print checl_plain($first_name); ?> <?php print $last_initial; ?></h4>
```

Comment:
Just made an [issue](https://github.com/DoSomething/dosomething/issues/4101) and assigned you. :)